.github/workflows/ci.yml

```diff
@@ -41,8 +41,4 @@ jobs:
           node-version: 12
       - run: npm install
       - run: npm test
-      - run: npm i codecov
-      - run: ./node_modules/.bin/codecov
-      - uses: codecov/codecov-action@v1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
+      - run: ./node_modules/.bin/c8 report --reporter=text-lcov | npx codecov@3 -t ${{ secrets.CODECOV_TOKEN }} --pipe
```

Comment:
I'm surprised you don't need to explicitly put the `with:` clause in here.  But if it works, 🤷‍♂

Reply:
It seems like we have access to the secrets set for the repo within the action. I think `with` is meant more for external actions, which specify the variables they expect using [inputs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputs).